### ISSUE TITLE
MouseSettings: Crashes when CursorHighlight was enabled

### ISSUE THREAD

**Issue by djwisdom** [SerenityOS/serenity#14200]
## Assertion failed.

Snapshot:

![image](https://user-images.githubusercontent.com/391659/172077254-13fd298a-5cbe-4972-b68e-ae89617e7b43.png)

**Comment by linusg** [comment]
cc @MacDue

**Comment by MacDue** [comment]
Were you doing anything in particular?

**Comment by MacDue** [comment]
It looks like that error happened because the cursor file for the preview could not be found, so it's likely due to your custom cursor theme.

**Comment by MacDue** [comment]
Yeah I only look for `/res/cursor-themes/{theme}/arrow.x2y2.png`, I guess I need to do something slightly smarter to handle various sized cursors
